newdle/client/src/components/ParticipantTable.module.scss

```diff
@@ -70,3 +73,27 @@
     }
   }
 }
+
+@media only screen and (max-width: 767px) {
+  .participant-table {
```

Comment:
do we need this selector? everything is inside `.participant-row` anyway

Reply:
Without it the `margin-bottom: 0` wasn't being applied as the value was taken from the normal css.. but this margin might not even be needed now with the latest changes in layout.

Reply:
Ah, but also other CSS seems to not get applied on mobile mode as well (flex-box related stuff)

Reply:
ah the joys of selector specificity :/